framework/platform/bpl/uci/db/bpl_db_uci.cpp

```diff
@@ -0,0 +1,93 @@
+#include "bpl_db_uci.h"
+
+
+namespace beerocks {
+namespace bpl {
+namespace db {
+
+bool uci_entry_exists(const std::string &config_file, const std::string &entry_type,
+                      const std::string &entry_name)
+{
+    //entry: config_file: entry_type(entry_name)
+    LOG(TRACE) << "uci_entry_exists() "
+               << "entry: " << config_file << ": " << entry_type << "(" << entry_name << ")";
+
+    return true;
+}
+
+bool uci_add_entry(const std::string &config_file, const std::string &entry_type,
```

Comment:
I don't think it makes sense to have this. Yes, uci.h has a separate uci_add_section function, but as a user of this library I will probably just want to do uci_set_entry() without worrying about adding the section if it doesn't exist yet.

Reply:
Yes, that's why in the upper layer (bpl_db) if the entry does not exists we add it.
But I believe that the "set" function by itself should not add a section if it does not exists

But you raised a good point and I reworked the "add" function to simply use "uci_set" instead of using "uci_add_section" & "uci_rename".

Reply:
And this also will add the functionally to change the type of the section if it already exists